CONTRIBUTING.md

```diff
@@ -56,12 +56,12 @@ Let's cover the developer flow for theses projects:
 
 ### Dashboard
```

Comment:
Maybe this part has to move the a dedicated CONTRIBUTING.md in the dashboard repo. WDYT ?

Reply:
separate issue? CC @sleshchenko

Reply:
https://github.com/eclipse/che/issues/16431